test/controllers/overseas.name.controller.spec.ts

```diff
@@ -55,10 +65,28 @@ mockGetApplicationData.mockReturnValue( APPLICATION_DATA_MOCK );
 const mockAuthenticationMiddleware = authentication as jest.Mock;
 mockAuthenticationMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
 
+const MOCKED_PAGE_URL = "/MOCKED_PAGE";
+
+const mockIsActiveFeature = isActiveFeature as jest.Mock;
+
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;
+mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
+
+const mockGetUrlWithParamsToPath = getUrlWithParamsToPath as jest.Mock;
+mockGetUrlWithParamsToPath.mockReturnValue(MOCKED_PAGE_URL);
+
+const mockGetUrlWithTransactionIdAndSubmissionId = getUrlWithTransactionIdAndSubmissionId as jest.Mock;
+mockGetUrlWithTransactionIdAndSubmissionId.mockReturnValue(MOCKED_PAGE_URL);
+
+const mockTransactionIdAndSubmissionIdExistInRequest = transactionIdAndSubmissionIdExistInRequest as jest.Mock;
+mockTransactionIdAndSubmissionIdExistInRequest.mockReturnValue(true);
+
 describe("Overseas Name controller", () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
+    mockIsActiveFeature.mockReset();
+    process.env.FEATURE_FLAG_ENABLE_REDIS_REMOVAL_27092023 = "false";
```

Comment:
process.env might be able to be removed here

Reply:
Done.